src/instructlab/dolomite/hf_models/modeling_utils/attention/__init__.py

```diff
@@ -9,8 +9,8 @@
 import torch
 
 # Local
-from ...config import CommonConfig
 from ...enums import AttentionHeadType
+from ...models.gpt_dolomite.config import GPTDolomiteConfig
```

Comment:
I think this might give circular import error due to stuff from this file being imported in the model class

Reply:
it was fine I have tested training the refactored codel

Reply:
<img width="778" alt="image" src="https://github.com/instructlab/GPTDolomite/assets/8325951/97b407f7-21c4-4c7f-aec5-93074c799ff2">

Reply:
But if you like, i have moved the config to a top-level `config.py` in https://github.com/instructlab/GPTDolomite/pull/3/commits/8afe81d60758e41ab00e1a98d4f86d3282e8b002